packages/ra-core/src/auth/usePermissions.spec.tsx

```diff
@@ -5,8 +5,11 @@ import { CoreAdminContext } from '../core/CoreAdminContext';
 
 import usePermissions from './usePermissions';
 
-const UsePermissions = ({ children, authParams }: any) => {
-    const res = usePermissions(authParams);
+const UsePermissions = ({ children }: any) => {
+    const authParams = {
+        retry: false,
+    };
+    const res = usePermissions({}, authParams);
```

Comment:
please don't name the variable `authParams` as it's misleading

Reply:
renamed